platform/broadcom/one-image.mk

```diff
@@ -21,6 +21,7 @@ $(SONIC_ONE_IMAGE)_LAZY_INSTALLS += $(DELL_S6000_PLATFORM_MODULE) \
                                $(ACCTON_AS7326_56X_PLATFORM_MODULE) \
                                $(ACCTON_AS7716_32XB_PLATFORM_MODULE) \
                                $(ACCTON_AS6712_32X_PLATFORM_MODULE) \
+                               $(ACCTON_AS7726_32X_PLATFORM_MODULE) \
```

Comment:
please do not change the file mode, keep it as 644. Please also check all other such file mode changes make sure they are in right mode.

Reply:
OK. I send "abdb2ff" commit to file mode to 644